### PR TITLE
teams/migration: a vote for hicqu as the reviewer

### PR DESCRIPTION
**Pull request by overvenus**
As title.

### CODE REVIEW

File: votes/0670-hicqu-reviewer.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+[@hicqu](https://github.com/hicqu) has been working on TiCDC for several months,
+he has made contributions to variety of the components of TiCDC
+(mysql-sink, schema-snapshot, sorter, etc).
+The details of his contributions are listed as follows:
+
+* [Authored pull requests](https://github.com/pingcap/tiflow/pulls?q=is%3Apr+author%3Ahicqu+is%3Aclosed).
+* [Reviewed pull requests](https://github.com/pingcap/tiflow/pulls?q=is%3Apr+reviewed-by%3Ahicqu).
+
+I (@overvenus) hereby nominate @hicqu as Tiflow Reviewer and call for a vote.
```
Comment (overvenus) [inline votes/0670-hicqu-reviewer.md:13]:
```suggestion
I (@overvenus) hereby nominate @hicqu as TiCDC Reviewer and call for a vote.
```

**Comment by amyangfei** [comment]
Please change file name to `votes/0676-hicqu-reviewer.md`

**Review by lance6716 — APPROVED** [review]

File: votes/0676-hicqu-reviewer.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Scope
+
+* Team Migration
+
+## Result
+
+Approved by X binding votes, 1 non-binding vote:
+
+* overvenus (non-binding)
```
Comment (overvenus) [inline votes/0676-hicqu-reviewer.md:27]:
```suggestion
* amyangfei (binding)
* lance6716 (binding)
* overvenus (non-binding)
```

File: votes/0676-hicqu-reviewer.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+The vote will be open for at least 3 days unless there is an objection or not enough votes.
+
+## Scope
+
+* Team Migration
+
+## Result
+
+Approved by X binding votes, 1 non-binding vote:
```
Comment (overvenus) [inline votes/0676-hicqu-reviewer.md:25]:
```suggestion
Approved by 2 binding votes, 1 non-binding vote:
```

**Pull request merged**